examples/em-server-multi-tier-cronjob/analysis/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # python 3
-FROM emission/e-mission-server.dev.server-only:4.0.0
+FROM agutt/e-mission-server:ceo_ebike_project_2022-09-19--45-23
```

Comment:
This is fine for now, but we should revert it after https://github.com/e-mission/e-mission-server/pull/875 and replace it with the e-mission-server image